src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -60,6 +60,9 @@ class HttpCache implements HttpKernelInterface, TerminableInterface
      *                            on responses that don't explicitly state whether the response is
      *                            public or private via a Cache-Control directive. (default: Authorization and Cookie)
      *
+     *   * skip_response_headers  Set of response headers that are never cached even if a response is cacheable (public).
```

Comment:
I think `skipped_response_headers` might be a better name for the option

Reply:
let's keep it as is, it also makes sense, is shorter and prevents reindenting all the docblock :P

Reply:
The docblock was the reason for me to keep it as short as it is now 🙈